src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -209,7 +209,19 @@ MOBILE_INTEGRATION_TEST_TOKEN = '{{ MOBILE_INTEGRATION_TEST_TOKEN }}'
 
 ####### For Constructing Global URLs ########
 
-LOCAL_APPS = ('dimagi.utils', 'gunicorn',)
+LOCAL_APPS = (
+    'dimagi.utils',
+    'gunicorn',
+    {% for entry in LOCAL_APPS.apps %}
+    {% if inventory_hostname in groups[entry.host] %}
+    '{{ entry.host }}'
+    '{{ entry.name }}''
```

Comment:
I think this should say `'{{ entry.name }}',` (replace last quote with comma)

Reply:
https://github.com/dimagi/commcare-cloud/pull/2248/commits/3c61731ed928debc39927158361acb12d7f957f5